Network/NetworkService.swift

```diff
@@ -63,4 +72,34 @@ class NetworkService {
     func cancel() {
         task?.cancel()
     }
+    
+    
+    //MARK: Private
+    private func makeQuery(for url: URL, params: [String: AnyObject], type: QueryType) -> NSMutableURLRequest? {
+        switch type {
+        case .JSON:
+            let mutableRequest = NSMutableURLRequest(url: url, cachePolicy: .reloadIgnoringLocalAndRemoteCacheData,
```

Comment:
You can extract cache policy and timeout interval before switch and use them in this switch.